lib/src/internal/interaction_endpoints.dart

```diff
@@ -378,15 +383,44 @@ class InteractionsEndpoints implements IInteractionsEndpoints {
     }
   }
 
-  /// Fetch the command permission overrides for a command in a guild.
   @override
   Future<SlashCommandPermissionOverrides> fetchCommandOverrides(Snowflake commandId, Snowflake guildId) async {
-    final response = await _client.httpEndpoints.sendRawRequest("/applications/${_client.appId}/guilds/$guildId/commands/$commandId/permissions", "GET");
+    try {
+      final response =
+          await _client.httpEndpoints.sendRawRequest("/applications/${_client.appId}/guilds/$guildId/commands/$commandId/permissions", "GET", auth: true);
+
+      return SlashCommandPermissionOverrides((response as IHttpResponseSucess).jsonBody as Map<String, dynamic>, _client);
+    } on IHttpResponseError catch (response) {
+      try {
+        // 10066 = Unknown application command permissions
+        // Means there are no overrides for this command... why is this an error, Discord?
+        if (jsonDecode(response.errorMessage)['code'] == 10066) {
+          return SlashCommandPermissionOverrides.empty(commandId, _client);
+        }
+      } on Exception {
+        // We got invalid JSON. The request is probably invalid, so we ignore it and return an error.
```

Comment:
can you add some kind of fine logging here. eating exception isnt that good of a practise